api/can_api_v2_definition.py

```diff
@@ -392,6 +395,11 @@ class Metrics(base_model.APIBaseModel):
         description="Ratio of staffed intensive care unit (ICU) beds that are currently in use.",
     )
 
+    bedsWithCovidPatientsRatio: Optional[float] = pydantic.Field(
+        ...,
+        description="Ratio of staffed hospital beds that are currently in use by COVID patients.",
```

Comment:
Can we add a note that this is calculated at the HSA level for counties?

Reply:
Ah, good idea

Reply:
aec817a283741576042293f6d812f257cf2c1dbd adds the copy "For counties, this is calculated using HSA-level data for the corresponding area."